res/style.css

```diff
@@ -78,7 +78,6 @@ body, #root, .profileViewer {
   left: 0;
   position: sticky;
   z-index: 2;
-  min-width: min-content;
```

Comment:
This value isn't known in Firefox, the right value in Firefox would be `-moz-min-content`. I tried using it and I found no difference, so I just removed it because, after all, that's the current behavior.